docs/index.rst

```diff
@@ -69,6 +69,35 @@ Example:
    to this mount.
 
 
+CDN
+~~~
+
+Crane can also be configured to support some advanced options when content resides
+on a Content Delivery Network. The URL match and replace parameters support rewriting
+the base paths of repos to a new location on the CDN.
+
+The URL authorization options allow Crane to append a query string authorization parameter
+based on a shared secret in order to allow content to be restricted on the CDN.
+
+The following options should go under a section named ``[cdn]``
+
+url_match
```

Comment:
let's try to avoid and re-phrase `base path`
maybe even worth showing a default example of what url you're talking about 
https://docs.pulpproject.org/plugins/pulp_docker/tech-reference/distributor.html#supported-keys like in redirect-url description key